manual/statements/built/README.md

```diff
@@ -1,5 +1,268 @@
+functions receive 2 arguments - the name of the column, and value to compare, but there
+are also variants that receive 2 iterables for columns & values separately.
+
+Note: as query are becoming more complex, repeating `QueryBuilder` at all places will make
+code less readable.  In this case you can import all (or only required) static functions
+for code simplification (this is rewritten example from above):
+
+```java
+import static com.datastax.driver.core.querybuilder.QueryBuilder.*;
+
+//...
+
+BuiltStatement selectOne = QueryBuilder.select().from("test", "test")
+        .where(eq("id", 1));
+```
+
+In case if you need to specify complex condition you can put additional clauses into chain
+of the `and` calls that accepts the same clauses as `where`:
+
+```java
+BuiltStatement select = QueryBuilder.select().from("test", "test")
+        .where(eq("id", "1")).and(eq("txt", "test"));
+```
+
+#### Other selection options
+
+For `SELECT` statements you can also specify a lot of different options: 
+ - `allowFiltering` generates corresponding `ALLOW FILTERING` part of query (***only use if you know what you're doing!***);
+ - `limit` & `perPartitionLimit` allows to specify amount of data to fetch;
+ - `groupBy` performs group of data;
+ - `orderBy` allows to specify sorting direction for specified clustering columns;
+ 
+This very "artificial" example shows the use for some of them:
+
+```java
+BuiltStatement selectOne = QueryBuilder.select().from("test")
+        .where(QueryBuilder.eq("id", 1)).limit(1).allowFiltering()
+        .perPartitionLimit(1).orderBy(desc("id"));
+```
+
+### Inserting data
+
+Insertion of data is straightforward - you're specifying the target table in call to
+`insertInto`, and then provide values to insert either by chaining the several calls to
+`value` function, or by using the `values` function passing the lists or arrays of column
+names & corresponding values. Following 2 examples are equivalent:
+
+```java
+QueryBuilder.insertInto("test").value("id", 4).value("t", "test 4");
+QueryBuilder.insertInto("test").values(Arrays.asList("id", "t"), Arrays.asList(4, "test 4"));
+```
+
+You can also insert JSON-formatted data by calling the `json` function & passing the data:
+
+```java
+QueryBuilder.insertInto("test").json("{\"id\":4, \"t\":\"test 4\"}");
+```
+
+`QueryBuilder` also allows to generate the statement that will use lightweight
+transactions (LWT) to check that inserted data doesn't exist yet.  You just need to add
+the call to `ifNotExists` to the statement:
+
+```java
+QueryBuilder.insertInto("test").value("id", 4).ifNotExists();
+```
+
+It also possible to specify additional metadata for inserted data, such as, TTL or
+timestamp.  This is achieved by using the `using` function and providing the `Using`
+object generated either by `ttl`, or `timestamp` functions of the `QueryBuilder` class. In
+case if you want to specify both of them, you need to wrap other into the call to `and`
```

Comment:
Suggestion: "If you want to specify both, you need to chain them together with the `and` operator".